<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>491</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* The tab widget frame */
	background-color: rgb(255, 255, 255);
border-top: 2px solid #2a2a2a; 
border-left: 2px solid #2a2a2a;
border-right: 2px solid #2a2a2a ; 
border-bottom: 2px solid #2a2a2a; 
background: #dcdcdc;
	
	
	/*border-top-color: rgb(0, 0, 0);*/

}
QTabWidget::tab-bar {
left: 0px; /* move to the right by 5px */
}
QTabBar::tab {
background: #dcdcdc;
border: 2px solid #535353;
border-bottom-color: #535353; 
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 80px;
padding: 8px;
color: rgb(50, 50, 162);
	color: rgb(0, 0, 0);
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:selected,QTabBar::tab:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #535353; /* same as pane color */
color: #dcdcdc;
font-size:13px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter contrat</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1231</width>
        <height>361</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MingLiU_HKSCS-ExtB&quot;;</string>
       </property>
       <property name="text">
        <string>Num_contrat :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="num_contrat">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>260</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="datecontrat">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MingLiU_HKSCS-ExtB&quot;;</string>
       </property>
       <property name="text">
        <string>Date contrat :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MingLiU_HKSCS-ExtB&quot;;</string>
       </property>
       <property name="text">
        <string>Montant :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="montant">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>70</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MingLiU_HKSCS-ExtB&quot;;</string>
       </property>
       <property name="text">
        <string>ID_EMP :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>140</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MingLiU_HKSCS-ExtB&quot;;</string>
       </property>
       <property name="text">
        <string>ID_CLIENT :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_emp">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_client">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher contrat</string>
     </attribute>
     <widget class="QTableView" name="tab_stock">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>741</width>
        <height>151</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Modification</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num_contrat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Montant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="montant2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="num_contrat2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>70</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="date_contrat2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date Cntrat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_EMP</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>10</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID_CLIENT</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_emp2">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_client2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>30</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Suppression</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;MingLiU_HKSCS-ExtB&quot;;</string>
        </property>
        <property name="text">
         <string>Num_contrat :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_sup_2">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>40</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Chercher</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>70</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>411</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>PDF</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>40</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>GENERER PDF</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>exel</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>30</y>
          <width>211</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>imprimer exel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Trie</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>131</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Par NUM_contrat</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>40</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Par date</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>40</y>
          <width>151</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Par montant</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>STAT</string>
     </attribute>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>biens</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>employees</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>clients</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>contrats</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>280</y>
        <width>321</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
